images/ubuntu/scripts/tests/Tools.Tests.ps1

```diff
@@ -144,7 +145,7 @@ Describe "Cmake" {
     }
 }
 
-Describe "erlang" -Skip:(Test-IsUbuntu22) {
+Describe "erlang" -Skip:(-not (Test-IsUbuntu20)) {
```

Comment:
Hello,
This test is not skipped in Ubuntu22 causing the following problem: `
==> azure-arm.build_image: /imagegeneration/SoftwareReport/Generate-SoftwareReport.ps1 : The term 'erl' is not recognized as a name of a cmdlet, function, script file, or executable program. ==> azure-arm.build_image: Check the spelling of the name, or if a path was included, verify that the path is correct and try again. ==> azure-arm.build_image: + CategoryInfo          : ObjectNotFound: (erl:String) [Generate-SoftwareReport.ps1], CommandNotFoundException ==> azure-arm.build_image: + FullyQualifiedErrorId : CommandNotFoundException,Generate-SoftwareReport.ps1`